cloudinit/handlers/cloud_config.py

```diff
@@ -139,6 +142,8 @@ def handle_part(self, data, ctype, filename, payload, frequency, headers):
             for i in ("\n", "\r", "\t"):
                 filename = filename.replace(i, " ")
             self.file_names.append(filename.strip())
+        except ValueError as err:
+            LOG.warning("Failed at merging in %s", err)
```

Comment:
Let's still report the filename here and the error because we could be dealing with separate #include cloud config part files and it'd be nice to know which one was None/empty
```suggestion
            LOG.warning("Failed at merging in cloud config part from %s: %s", filename, err)
```

Reply:
Agreed